onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -139,7 +137,9 @@ def create_backend_test(testname=None):
                                       '^test_reduce_[a-z1-9_]*_negative_axes_.*',
                                       'test_squeeze_negative_axes_cpu',
                                       'test_unsqueeze_negative_axes_cpu',
-                                      'test_constant_pad_cpu']
+                                      'test_constant_pad_cpu',
+                                      'test_edge_pad_cpu',
```

Comment:
(FYI comment) Added to the NGraph specific exclusion list as NGraph doesn't support opset-11 yet.